usaspending_api/disaster/tests/integration/test_disaster_agency_count.py

```diff
@@ -26,6 +26,14 @@ def test_agency_count_success(client, monkeypatch, disaster_account_data, helper
     assert resp.data["count"] == 0
 
 
+@pytest.mark.django_db
+def test_agency_count_with_award_types(client, monkeypatch, faba_with_toptier_agencies, helpers):
```

Comment:
If you have time I would advise that you add an additional Award to your test data. Right now the test data only has two Awards if I am following correctly so having a count of 2 could be a false positive.